@navikt/ds-react/util/src/uuid.ts

```diff
@@ -0,0 +1,3 @@
+import { v4 as uuidv4 } from "uuid";
+
+export const uuid = () => uuidv4();
```

Comment:
Jeg tenker det er enklere at de komponentene som har bruk for uuid bare bruker biblioteket direkte.